lib/polyfill/patchedmediakeys_apple.js

```diff
@@ -25,6 +25,25 @@ goog.require('shaka.util.StringUtils');
  * @export
  */
 shaka.polyfill.PatchedMediaKeysApple = class {
+  /**
+   * Installs the polyfill if needed.
+   */
+  static defaultInstall() {
+    if (!window.HTMLVideoElement || !window.WebKitMediaKeys) {
+      // No HTML5 video or no prefixed EME.
+      return;
+    }
+    if (navigator.requestMediaKeySystemAccess &&
+         // eslint-disable-next-line no-restricted-syntax
+         MediaKeySystemAccess.prototype.getConfiguration) {
+      // Unprefixed EME available
+      return;
+    }
+    // If there is no unprefixed EME and prefixed EME exists, apply installation
+    // by default. Eg: older versions of Safari.
```

Comment:
Is there any realistic usage of these versions?  Judging by https://caniuse.com/?search=MediaKeys and https://caniuse.com/usage-table, that would be Safari 12.0 and under, which have virtually no usage.

If your company has users on these older versions, I'm not against this, but I do wonder if this is maybe not useful.

Reply:
I don't really know how many people use it, but I do use a device running iOS 10 to test that nothing has broken from time to time.